### ISSUE TITLE
[example-app-base] Main landmark is not defined 

### ISSUE THREAD

**Issue by lucas-stanford** [microsoft/fluentui#18049]
### Environment Information
n/a

### Describe the issue:
If main landmark is not defined on the page than screen reader not able to navigate on the page in landmark mode.

### Please provide a reproduction of the issue in a codepen:
https://fabricweb.z5.web.core.windows.net/oufr/7.170.0/

#### Actual behavior:
Main landmark is not defined on the page.

#### Expected behavior:

Main landmark should be defined on the page. 

Repro steps:

1. Open URL:https://fabricweb.z5.web.core.windows.net/oufr/7.170.0/
2. Navigate to left pane and select it, navigate to 'Button' and select it.
3. Open accessibility insight for web and verify there is no 'main' landmark.

### Documentation describing expected behavior
https://www.w3.org/TR/wai-aria-practices/examples/landmarks/main.html

---
Bug id: 9551863

**Comment by tyveille** [comment]
@TristanWatanabe 

Are there any updates for this issue?
